README.md

```diff
@@ -4,6 +4,14 @@
   </a>
 </p>
 
+<div align="center">
+
+**[Web Site](https://vuepress.vuejs.org/) |
+[Latest Guide](https://vuepress.vuejs.org/guide/) |
+[GitHub](https://github.com/vuejs/vuepress/)**
```

Comment:
`Github` is duplicate.

Reply:
It is functional when the readme is used in another page as **NPM**

Reply:
Hmmm... npm has supported for that.

Reply:
So what do you think about adding other important links like Config Reference, Default Theme Config?